### PR TITLE
Anonymizes data in Rogue so that we're compliant with GDPR

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
This PR anonymizes data in Rogue so that we're compliant with GDPR:
- Changes all EU member signups' `why_participated` to be `EU Member. Removed because of GDPR`.
- Changes all EU member posts' `text` to be `EU Member. Removed because of GDPR`.
- Deletes all EU member posts' image URLs from s3 and changes them to `NULL` in the database. 
- Tags all EU member posts as "Hide In Gallery" 
- All of the above is changed for all records, even if they have been soft deleted. 

#### How should this be reviewed?
👀 
Did I miss anything that we need to do here? 

#### Relevant tickets
Fixes [this](https://www.pivotaltracker.com/n/projects/2019429/stories/157709055) Pivotal Card.

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md

### CODE REVIEW

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -25,6 +25,7 @@ class Kernel extends ConsoleKernel
         Commands\ReviewSignup::class,
         Commands\SendToQuasar::class,
         Commands\ForceDeleteTestRecords::class,
+        Commands\GDPRComplianceCommand::class,
```
Comment (DFurnes) [inline app/Console/Kernel.php]: Totally unrelated to this pull request, but we can have Laravel auto-load these (see [Northstar](https://github.com/DoSomething/northstar/blob/742e3de1d46621926b540cd2052456e08d500e70/app/Console/Kernel.php#L20-L29)'s kernel).

File: app/Console/Commands/GDPRComplianceCommand.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        $temp = tempnam('temp', 'command_csv');
+        file_put_contents($temp, fopen($this->argument('path'), 'r'));
+
+        // Load users from the CSV
+        $users_csv = Reader::createFromPath($temp, 'r');
+        $users_csv->setHeaderOffset(0);
+        $users = $users_csv->getRecords();
+
+        // Anonymize user data.
+        $this->info('rogue:GDPRcompliance: Anonymizing data...');
```
Comment (DFurnes) [inline app/Console/Commands/GDPRComplianceCommand.php]: Nitpicky, let's use `rogue:gdpr` here like our other log lines.

Comment (chloealee) [reply app/Console/Commands/GDPRComplianceCommand.php]: ah yes - I shortened it and forgot to change there! Thanks for pointing that out!

File: app/Console/Kernel.php (hunk 1 above)
Comment (chloealee) [reply app/Console/Kernel.php]: Yes, I love this!!

**Pull request merged**